tests/bp/test_infer.py

```diff
@@ -0,0 +1,119 @@
+from itertools import product
+
+import numpy as np
+
+from pgmax.bp import infer
+
+print("We assume that factor_id is increasing and take values in 0,..., n_factors")
+
+
+def test_pass_fac_to_var_messages():
```

Comment:
Fine to leave it as is for now. Once we added proper interface for OR factors we should reimplement this unit test using automated derived inference functions from a factor graph specified through the interface.